site/content@beta/agent-chat-rtm-api/scopes.md

```diff
@@ -0,0 +1,114 @@
+---
+weight: 20
+---
+
+# Scopes
+
+Scopes define the access to certain resources.
+
+- `ro` means read only
+- `rw` means read/write
+- all `rw` scopes contain `ro` privileges
+- all `access` scopes contain `my` privileges
+- all `all` scopes contain `access` privileges and `my` privileges
+
+
+## Accessing chat 
+
+This chart shows every possible chats scenarios:
+
+- chats without my access and without my presence (`chats -A -P`)
+- chats with my access but without my presence (`chats +A -P`)
+- chats with my presence but without my access (`chats -A +P`)
+- chats with my access and with my presence (`chats +A +P`)
+
+![](./chats.png)
+
+The table shows scopes dependency of accessing chat:
+
+|          | `chats -A -P` | `chats +A -P` | `chats -A +P` | `chats +A +P` |
+| -------- | ------------- | ------------- | ------------- | ------------- |
+| `all`    | ✓             | ✓             | ✓             | ✓             |
+| `access` | -             | ✓             | ✓             | ✓             |
+| `my`     | -             | -             | ✓             | ✓             |
+
+## Accessing parts of chat 
+
+The table shows scopes dependency of accessing chat parts:
+
+|                           | meta data | conversation data |
+| ------------------------- | --------- | ----------------- |
+| `chats-*:rw`              | rw        | rw                |
+| `chats.conversation-*:rw` | ro        | rw                |
+| `chats-*:ro`              | ro        | ro                |
+
+## Scope types
+
+#### Agents scopes
+
+| Scope            | role          | Description                                            |
+| ---------------- | ------------- | ------------------------------------------------------ |
+| `agents--my:rw`  | normal        | Write permission for my profile configuration          |
+| `agents--all:rw` | administrator | Write permission for all agents profiles configuration |
+| `access_rules:ro` | administrator | Read permission for auto chat scopes configuration       |
+| `access_rules:rw` | administrator | Read/write permission for auto chat scopes configuration |
+
```

Comment:
Czy ja tu czegoś nie kumam czy to jest zle wytłumaczone? Czemu access_rules dajemy do Agents scopes?
Co to znaczy "auto chat scopes configuration"?